### PR TITLE
added API requests, routes, FAQ, contact, loading

### PR DESCRIPTION
**Pull request by MariiaTyshkovets**
added API requests, routes, FAQ, contact, loading

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Contact from './components/Contact/Contact';
+import FAQ from './components/FAQ/FAQ';
+import NotFound from './components/NotFound/NotFound';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <Routes>
+        <Route path="/mariia_app" element={<Content />}/>
```
Comment (roskyba) [inline src/App.js:16]: Generally, routing for app with your name is not good :) 
Remove `mariia_app`

File: src/components/Tr/Tr.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Tr(props) {
```
Comment (roskyba) [inline src/components/Tr/Tr.js]: Give better names for components - `Tr` is not a good choice

Comment (roskyba) [reply src/components/Tr/Tr.js]: At least - `TableRow` will be better

File: src/components/Tr/Tr.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    return (
+        <tr>
+            <td>{props.index + 1}</td>
+            <td>{props.obj.orderName}</td>
+            <td>{props.obj.orderPhone}</td>
+            <td>{props.obj.orderFast ? 'YES': 'NO'}</td>
+            <td>{props.obj.orderPrice}</td>
+            <td>{Object.entries(props.obj.orderProducts).map((obj, index, arr) => {
+                    if (index !== arr.length - 1) {
+                        return `${obj[0]}: ${obj[1]}, `
```
Comment (roskyba) [inline src/components/Tr/Tr.js]: O_o

File: src/components/Tr/Tr.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+                    } else {
+                        return `${obj[0]}: ${obj[1]}`
```
Comment (roskyba) [inline src/components/Tr/Tr.js]: o_O

same as 14 line

what's the point to have this `if...else` statement then?

File: src/containers/Content/index.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+            ingredients: {},
+            orderedList: [],
+            total: "1.00",
+            isModalActive: false,
+            isPurchaseActive: false,
+            isLoading: true
+        }
+        
+        this.reload.bind(this);
+        this.addIngredientBuilder.bind(this);
```
Comment (roskyba) [inline src/containers/Content/index.js:25]: Arrow functions will save you from bindings ^_^

File: src/containers/Content/index.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    changeActiveModal = () => {
+        this.setState((oldState) => {
+            return {
+                ...oldState,
+                isModalActive: !oldState.isModalActive
+            }
+        });
+    }
+
+    addToBill = () => {
```
Comment (roskyba) [inline src/containers/Content/index.js:94]: But you have it as an arrow function. No need to `.bind` it inside constructor

File: src/containers/Content/index.js [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+        let li = "";
+        if (this.state.total > 0) {
+            Object.entries(this.state.ingredients).forEach(([key, value]) => {
+                let price = this.state.orders.filter((item) => item.ingredient === key)[0].price.toFixed(2);
+                if (value > 0) {
+                    li = [li, (<li key={key}>{key}: {value} x {price} $</li>)]
```
Comment (roskyba) [inline src/containers/Content/index.js:100]: Complicated logic and hardly maintainable code. Make it simpler. Avoid using loops inside loops

File: src/containers/Content/index.js [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+        return li;
+    }
+
+    addIngredientBuilder = () => {
+        let block = "";
+        this.state.orderedList.forEach((ingre, index) => {
+            let nameOfClass = ingre.toLowerCase();
+            block = [block, (<div className={nameOfClass + " ingredient"} key={nameOfClass + "_" + index}></div>)]  
+        });
+        return block;
```
Comment (roskyba) [inline src/containers/Content/index.js:113]: This function needs refactoring

**Review by roskyba — APPROVED** [review]
Keep improving with React!

`Points for MR - 86/100`

File: src/components/Tr/Tr.js (hunk 4 above)
Comment (MariiaTyshkovets) [reply src/components/Tr/Tr.js]: they aren`t same. The last element should be without comma in the end

**Pull request merged**